core/src/main/thrift/tabletserver.thrift

```diff
@@ -265,6 +265,8 @@ service TabletScanClientService {
     1:client.ThriftSecurityException sec
     2:TSampleNotPresentException tsnpe
     3:ScanServerBusyException ssbe
+    4:NotServingTabletException nste
+    5:TooManyFilesException tmfe    
```

Comment:
One of the expectations in a patch release is that we can do rolling upgrades, and clients and servers may be upgraded at different times because patch releases are both forwards and backwards compatible. These are both already TExceptions, which should be caught in the generated code as a generic catch-all. However, I'm not sure what the exact behavior would be when these are declared explicitly in 2.1.1, but not in 2.1.0 if a client and server of different versions are talking to each other.

Before approving this, I would like to know that the behavior is sensible if there is a 2.1.0 client talking to a 2.1.1 server, or a 2.1.1 client talking to a 2.1.0 server when these exceptions are thrown. Have you tested these scenarios? What is the impact?

Reply:
Rebased branch to main, so this is OBE now